source/about.rst

```diff
@@ -296,4 +296,4 @@ Which Components are used in Clear Linux?
 .. _Supplied Bundles: https://clearlinux.org/software
 .. _micro-config-drive: https://github.com/clearlinux/micro-config-drive
 .. _Telemetrics: https://github.com/clearlinux/telemetrics-backend
-.. _packages source code: https://github.com/clearlinux-pkgs/
\ No newline at end of file
+.. _packages source code: https://github.com/clearlinux-pkgs/
```

Comment:
Add 

```
.. _Clear_linux_store: https://clearlinux.org/software

```

Reply:
It is already there, there was no space between "see" and the link in the original copy causing it to not get picked up. Fixed by adding the space.